go-tools/lib/lib.go

```diff
@@ -0,0 +1,164 @@
+package lib
+
+import (
+	"encoding/hex"
+	"flag"
+	"fmt"
+	mqtt "github.com/eclipse/paho.mqtt.golang"
+	. "github.com/iotaledger/iota.go/v2"
+	ed "github.com/iotaledger/iota.go/v2/ed25519"
+)
+
+const (
+	ApiPort   = 14265
+	MqttPort  = 1883
+	LocalHost = "127.0.0.1"
+)
+
+var (
+	BalanceSeed   = []byte{0xde, 0xad, 0xbe, 0xef}
+	NullMessageId = MessageID{}
+)
+
+func Must(err error) {
+	if err != nil {
+		panic(err)
+	}
+}
+
+func ObtainAPI(nodeUrl string, apiPort int) (*NodeAPIClient, *NodeInfoResponse) {
+	endpoint := flag.String("endpoint", fmt.Sprintf("http://%s:%d", nodeUrl, apiPort), "endpoint")
```

Comment:
This flag shouldn't be in this function. The caller should be set up/use the flag.